db/src/debug.rs

```diff
@@ -21,13 +21,20 @@ use rusqlite::types::{ToSql};
 use tabwriter::TabWriter;
 
 use bootstrap;
+use db::TypedSQLValue;
```

Comment:
No, these are all modules.  Or do you want all `use module;` before any `use module::symbol`?  I've been interleaving them.